src/status_im/multiaccounts/login/core.cljs

```diff
@@ -1,48 +1,48 @@
 (ns status-im.multiaccounts.login.core
-  (:require [clojure.string :as string]
-            [re-frame.core :as re-frame]
-            [status-im.async-storage.core :as async-storage]
-            [status-im.chat.models.link-preview :as link-preview]
-            [status-im.communities.core :as communities]
+  (:require [re-frame.core :as re-frame]
```

Comment:
could you elaborate on this format change?

Reply:
It's unneeded .. After fixing the sync issue I am going to clean up this PR